File: app/src/main/java/com/apache/fastandroid/support/bean/UserBean.java
package com.apache.fastandroid.support.bean;

import java.io.Serializable;

/**
 * Created by jerryliu on 2017/7/9.
 */

public class UserBean implements Serializable {
    public String userName;
    public String password;


    public UserBean(String userName, String password) {
        this.userName = userName;
        this.password = password;
    }
}

File: app/src/main/java/com/apache/fastandroid/demo/bean/UserBean.kt
package com.apache.fastandroid.demo.bean

import androidx.databinding.BaseObservable
import androidx.databinding.Observable
import androidx.databinding.ObservableField
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.*

/**
 * Created by Jerry on 2021/1/22.
 */

data class UserBean( var name:String?,  var age:Int = 10):Serializable
